__tests__/hooks/useGroupInfo.test.js

```diff
@@ -33,8 +33,14 @@ jest.mock('recoil', () => ({
   atom: jest.fn(),
   useRecoilState: jest.fn(() => [{}, jest.fn()])
 }));
+jest.mock('react-native-compressor', () => {
+  return {
+    compress: jest.fn(() => 'file:///imag.jpg')
+  };
+});
 
 describe('useGroupInfo should run correctly', () => {
+  afterEach(cleanup);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `cleanup` function from `@testing-library/react-hooks` is now being called in the `afterEach` block. This is a good practice as it ensures that after each test, any setup done for the test is properly cleaned up to avoid any side effects that could affect other tests.
```diff
+ afterEach(cleanup);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->